hail/python/hail/experimental/vcf_combiner/__init__.py

```diff
@@ -0,0 +1,14 @@
+from .vcf_combiner import run_combiner, transform_gvcf, transform_one, parse_as_fields
+from .sparse_split_multi import sparse_split_multi
+from .sparse_mt_utils import lgt_to_gt
+from .densify import densify
+
+__all__ = [
+    'run_combiner',
+    'transform_gvcf',
+    'transform_one',
```

Comment:
Need to add `combine_gvcfs` here too.

Reply:
combine_gvcfs shouldn't be one of the user-exposed functions -- it should be totally internal to `vcf_combiner.py`. I'll build a replacement that can do incremental merges, but that will come in a separate PR.

Reply:
That's reasonable. The config adds a lot of the functionality that wasn't being provided to users regarding merge plans. LGTM.